2nd-gen/packages/swc/.storybook/DocumentTemplate.mdx

```diff
@@ -18,33 +18,14 @@ import '@spectrum-web-components/tabs/sp-tab-panel.js';
 
 <Title />
 <Subtitle />
+<Primary />
 
-<sp-tabs id="component-doc-tabs" size="xl" selected="overview" auto label="Component Documentation">
-    <sp-tab value="overview">Overview</sp-tab>
-    <sp-tab value="usage">Usage</sp-tab>
-    <sp-tab value="accessibility">Accessibility</sp-tab>
-    <sp-tab value="api">API</sp-tab>
-    <sp-tab value="examples">Examples</sp-tab>
-    <sp-tab-panel value="overview" style={{ flexDirection: 'column' }}>
-        <Description />
-        <Primary />
-        <Controls />
-    </sp-tab-panel>
-    <sp-tab-panel value="usage" style={{ flexDirection: 'column' }}>
-        <SpectrumDocs tag="usage" />
-    </sp-tab-panel>
-    <sp-tab-panel value="accessibility" style={{ flexDirection: 'column' }}>
-        <SpectrumDocs tag="a11y" />
-    </sp-tab-panel>
-    <sp-tab-panel value="api" style={{ flexDirection: 'column' }}>
-        <ArgTypes />
-    </sp-tab-panel>
-    <sp-tab-panel value="examples" style={{ flexDirection: 'column' }}>
-        <SpectrumDocs tag="examples" />
-        <Stories />
-    </sp-tab-panel>
+<SpectrumDocs tag="usage" />
 
-</sp-tabs>
+## API
+
+<Primary />
+<Controls />
```

Comment:
I like being able to play with the API right from the docs page to understand it better.

Reply:
so if we have two primary stories these controls will effect both of them. maybe i can create a static story block for the first story render at the top of overview...